lib/active_snapshot.rb

```diff
@@ -16,6 +16,7 @@ def self.config(&block)
 end
 
 ActiveSupport.on_load(:active_record) do
+  require "active_snapshot/models/concerns/diffable"
```

Comment:
I know that I previously suggested making this into a concern but I think we've simplified it enough that I'd rather have it directly in the class 🙏

Reply:
ok, moved it into the class!